napari/layers/utils/string_encoding.py

```diff
@@ -173,18 +165,14 @@ class FormatStringEncoding(_DerivedStyleEncoding[StringValue, StringArray]):
     encoding_type: Literal['FormatStringEncoding'] = 'FormatStringEncoding'
 
     def __call__(self, features: Any) -> StringArray:
+        feature_names = features.columns.to_list()
         values = [
-            self.format.format(**_get_feature_row(features, i))
-            for i in range(len(features))
+            self.format.format(**dict(zip(feature_names, tuple_)))
+            for tuple_ in features.itertuples(index=False, name=None)
```

Comment:
When possible use variable names to describe what something represents, not what type it is. Here, `itertuples` gives us an iterator over named tuples that represent rows of the dataframe, so let's use `row`?

```suggestion
            self.format.format(**dict(zip(feature_names, row)))
            for row in features.itertuples(index=False, name=None)
```

Reply:
Sure!